cmd/jenkins-usage-stats/fetch.go

```diff
@@ -28,7 +28,7 @@ func NewFetchCmd(ctx context.Context) *cobra.Command {
 	cobraCmd := &cobra.Command{
 		Use:   "fetch",
 		Short: "Fetch raw usage data from Azure",
-		Run: func(cmd *cobra.Command, args []string) {
```

Comment:
not 100% sure on if this is a great idea, its much clearer what the args could be used with the old name. Although not too keen on disabling the rule